src/engine/ruleset/decoders/fortinet/firewall.yml

```diff
@@ -0,0 +1,540 @@
+---
+name: decoder/fortinet-firewall/0
+
+metadata:
+  title: Decoder for parsing fortinet firewall logs
+  description: Decoder for parsing fortinet firewall logs
+  compatibility: >
+    Wazuh Engine
+  author:
+    name: Wazuh, Inc.
+    date: 2023/01/10
+  references:
+    - https://docs.fortinet.com/document/forticlient/7.0.2/log-message-reference/327109
+
+sources:
```

Comment:
Ditto Question-desing: Did you try to see if it goes through the queue-syslog decoder first? If it goes through this it is sure that it goes through the queue-localfile so I suppose so. Please add queue-syslog to the source